### DISCUSSION TITLE
演示项目怎么进聊天界面啊？

### DISCUSSION THREAD

**Discussion by elarbor** [Evansy/MallChatWeb#116] [Q&A]
打开演示项目只能在首页，好像soul的首页。但是怎么打开聊天界面呢？

**Comment by Evansy** [comment]
如果你本地安装完依赖并 `pnpm dev` 启动后的话, 浏览器访问 `http://localhost:9988` 就可以看到了呀, 你也可以访问 `https://mallchat.cn` 体验
